pkgs/fabric.yaml

```diff
@@ -4,5 +4,16 @@ dependencies:
   run: [paramiko]
 
 sources:
-  - url: https://pypi.python.org/packages/source/F/Fabric/Fabric-1.8.0.tar.gz
-    key: tar.gz:ab3mo2v654775els4euq43sipyr2oiu3
+- key: tar.gz:mzvd3kjconjs2osj3unmkpr5oiof5qxs
+  url: https://pypi.python.org/packages/source/F/Fabric/Fabric-1.10.2.tar.gz
+
+build_stages:
+
+- name: install
+  after: setup_dirs
+  handler: bash
+  bash: |
```

Comment:
huh, weird.  maybe put a comment in the file explaining what's going on for the next person who looks at this.